src/PhpPact/Consumer/Matcher/Matcher.php

```diff
@@ -660,4 +660,21 @@ public function contentType(string $contentType): array
             'pact:matcher:type' => 'contentType',
         ];
     }
+
+    /**
+     * Allows defining matching rules to apply to the keys in a map
+     *
+     * @param array<string, mixed> $values
+     * @param array<mixed>         $rules
+     *
+     * @return array<string, mixed>
+     */
+    public function eachKey(array $values, array $rules): array
+    {
+        return [
+            'rules' =>             $rules,
+            'value'             => $values,
+            'pact:matcher:type' => 'eachKey',
```

Comment:
Nitpick: The `=>` aren't aligned. Up to you whether you want to fix this before merging or not, as it is purely stylistic 👍

Reply:
I fixed it. Thanks for pointing it out.